pkg/local_object_storage/metabase/counters.go

```diff
@@ -0,0 +1,159 @@
+package meta
+
+import (
+	"encoding/binary"
+	"fmt"
+
+	cid "github.com/nspcc-dev/neofs-sdk-go/container/id"
+	oid "github.com/nspcc-dev/neofs-sdk-go/object/id"
+	"go.etcd.io/bbolt"
+)
+
+var objectPhyCounterKey = []byte("phy_counter")
+var objectLogicCounterKey = []byte("logic_counter")
```

Comment:
Can we leave the file in place? It is hard to see the actual diff because of moving.

Reply:
Also, `VERSION.md` could be updated.

Reply:
> Can we leave the file in place?

ok, renamed it back

> VERSION.md could be updated.

updated